_posts/2019-09-13-git-help-desk.md

```diff
@@ -0,0 +1,17 @@
+---
+author: Nic James
+category: General
+date: 2019-09-13 06:00:00
+layout: post
+image: /static/img/blog/git-help-desk.png
+title: "Introducing the Git Help Desk!"
+---
+
+We are excited to continue making sprints more accessible to everyone this year! In order to make it easier for the sprint leaders to focus on sprinting, we are setting up a **Git Help Desk** during both days of the sprints to help sprinters with their commits. You don’t have to be an expert, but if you are willing to pair program with someone until they find their solution, please sign up to be a part of the Git Help Desk!
+
+If you’re interested in helping out at DjangoCon US, please consider [volunteering for the help desk](https://docs.google.com/spreadsheets/d/1VO8Xh6xPOQb32WjUD7pKJCMXvqCdcMz95CqIoNToUsA/edit#gid=164602686).
+
+For the sprint newbies or anyone who wants a refresher, we will also be holding an **Introduction to Contributing to Open Source** in Cabrillo 2 from 9am-12pm on Thursday, Sept. 26 hosted by Jeffrey Eliasen.
+
+Already know how to use Git and GitHub but need help getting set up to contribute to Django? This year we will have Carlton Gibson, one of our Django Fellows, walking you through **how to start contributing to Django**, also in Cabrillo 2 from 9am-12pm on Thursday, Sept. 26.
```

Comment:
One talk is title case and one is sentence case. Which is correct?

Reply:
The first one looks like a formal title. The second looks more like general assistance.